package-lock.json

```diff
@@ -1,8 +1,415 @@
 {
 	"name": "copyright-inserter",
-	"version": "1.0.12",
-	"lockfileVersion": 1,
+	"version": "1.0.13",
+	"lockfileVersion": 2,
```

Comment:
what is a reason for upgrading the locker version?

Reply:
That happened automatically due to me using a newer version of NPM - I'll revert and fix that.